service/filter.go

```diff
@@ -68,9 +70,9 @@ func ParseRequestForm() Filter {
 		ct, err := ContentType(req)
 		if err == nil {
 			switch ct {
-			case "application/x-www-form-urlencoded":
+			case definition.MIMEURLEncoded:
 				err = req.ParseForm()
-			case "multipart/form-data":
+			case definition.MIMEFormData:
 				err = req.ParseMultipartForm(32 << 20)
```

Comment:
Is it safe to do so?

Reply:
It's safe. The value `32 << 20` is default and fixed value in `http` package. If body is larger than 32M, `ParseMultipartForm` would cache it in local file system.